app/grandchallenge/emails/admin.py

```diff
@@ -23,9 +25,17 @@ def schedule_emails(modeladmin, queryset, request, action):
         )
 
 
+class EmailAdminForm(ModelForm):
+    class Meta:
+        model = Email
+        widgets = {"body": MarkdownEditorAdminWidget}

```

Comment:
Does the email render correctly in the preview or do we need a separate view?

Reply:
The content that we enter there renders correctly yes. You don't see the final email there though, because the content is only added to the template later. 

The markdown editor in general (we use it also for blog posts) doesn't look good in the admin though. I'm not sure how easy it is to improve, but in any case that would be something for a separate PR.

![image](https://user-images.githubusercontent.com/30069334/163403758-fe11cdaf-eb04-401b-950b-f10a7d23f703.png)